azure-pipelines-wrapper/conflict_detect.js

```diff
@@ -12,8 +12,8 @@ const SUCCESS = 'success'
 
 async function check_create(app, context, uuid, owner, repo, commit, check_name, result, status, output_title, output_summary){
     if (result == '') {
-        app.log.info(`[ CONFLICT DETECT ] [${uuid}] check_create: result=BLANK`)
-        return
+        app.log.error(`[ CONFLICT DETECT ] [${uuid}] check_create: result=BLANK`)
+        result = SUCCESS
```

Comment:
Why is the result SUCCESS when there is an error above?

Reply:
It will show result on sonic-buildimage checkers. It is microsoft private checker.
We only mark PR as failure with high confidence. When result is blank, maybe it is service error.